src/api/spec/features/beta/webui/users/notifications_controller_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'browser_helper'
+
+RSpec.describe 'User notifications', type: :feature, js: true do
+  let!(:user) { create(:confirmed_user, :in_beta) }
+
+  describe 'when having no notifications' do
+    context 'when accessing the notifications page' do
+      before do
+        login user
+        visit my_notifications_path
+      end
+
+      it 'shows no notifications' do
+        expect(page).to have_text('There are no notifications')
+      end
+    end
+  end
+
+  describe 'when having notifications' do
+    let!(:notification_for_projects_comment) { create(:web_notification, :comment_for_package, subscriber: user) }
+    let!(:another_notification_for_projects_comment) { create(:web_notification, :comment_for_package, subscriber: user) }
+    let(:notifiable) { notification_for_projects_comment.notifiable }
+    let(:another_notifiable) { another_notification_for_projects_comment.notifiable }
+    let(:project) { notifiable.commentable.project }
+
+    context 'when clicking on the Comments filter' do
+      before do
+        login user
+        visit my_notifications_path
+      end
+
+      it 'shows all unread comment notifications' do
+        find(id: '#notifications-dropdown-trigger').click if mobile?
```

Comment:
To be consistent with other feature specs: 
```suggestion
        find('#notifications-dropdown-trigger').click if mobile?
```